### ISSUE TITLE
Expectations for standards-positions with only an explainer (pre-spec)

### ISSUE THREAD

**Issue by zcorpan** [mozilla/standards-positions#808]
Context: https://github.com/mozilla/standards-positions/issues/806#issuecomment-1562939075

Requesting standards-position before a spec exists i OK and encouraged, so that we can engage sooner. However we may need to update documentation about this to clarify what is expected for both the reporter and for Mozilla for such issues.

**Comment by dveditz** [comment]
There's nothing wrong with private feedback on a new proposal, but our standards position process doesn't seem like the place for it. Technically feedback here is given in public so it's not private feedback, but the audience is narrow so it's not functionally a "public" proposal either. I'd be more comfortable using our "standards" positions to give feedback on proposals that have at least been presented for discussion in one of the standards groups we participate in. Benefits:
* chance for feedback from people who don't think like browser-engine developers (web app and server devs)
* more clarity on IP rules for the proposal

**Comment by martinthomson** [comment]
I agree with Dan. It is perfectly fine to ask for input on just about anything here. But we might decline to engage or defer engagement for work that is clearly incomplete. Coming here might be a good way to make us aware of something when it is very early, but it is better in every respect to discuss work in standards groups. If not immediately, then as soon as possible.
